Tests/ViewInspectorTests/InspectorTests.swift

```diff
@@ -64,6 +64,10 @@ final class InspectorTests: XCTestCase {
         XCTAssertEqual(name1, "Struct3<Int>")
         let name2 = Inspector.typeName(type: Struct1.self)
         XCTAssertEqual(name2, "Struct1")
+        let name3 = Inspector.typeName(
```

Comment:
This type is similar to the type that I am having issues with in https://github.com/nalexn/ViewInspector/pull/247#discussion_r1229999348